tests/functional/test_splinter_shadow.c

```diff
@@ -323,25 +319,25 @@ test_splinter_shadow_iterate_tree(AvlTreeLinks               *root,
  *       Will allocate memory for the key and value arrays.
  *-----------------------------------------------------------------------------
  */
-
 platform_status
 test_splinter_build_shadow_array(test_splinter_shadow_tree  *tree,
                                  test_splinter_shadow_array *shadow_array,
-                                 platform_heap_handle        hh)
+                                 platform_heap_handle        hid)
```

Comment:
Looks like `hid` is unused now.

Reply:
Took it out. I did some more cleanup to percolate up the stack, cleaning up fn-args like `hh', `hid` that are no longer needed. Some might have slipped thru ... I'll make another pass once this PR checks-in.